packages/fxa-settings/src/lib/gql.ts

```diff
@@ -45,9 +41,12 @@ export function createApolloClient(gqlServerUri: string) {
   // errorLink handles error responses from the server
   const errorLink = onError(errorHandler);
 
+  // uploadLink makes the actual requests to the server(httpLink with uploads)
+  const uploadLink = createUploadLink({ uri: `${gqlServerUri}/graphql` });
+
   const apolloClient = new ApolloClient({
     cache,
-    link: from([errorLink, authLink, httpLink]),
+    link: from([errorLink, authLink, uploadLink as any]),
```

Comment:
@dannycoates This is the unfortunate place where I believe we had mismatches, though it looks like we ended up forcing it by casting to `any` anyways. So maybe we don't need to downgrade the `apollo-client`?

If there's some way to get the right object version here without the cast to any, and fixup the version deps involved, that'd definitely be preferable to having to use this kludge. Perhaps we should at least flag the object version mismatch here so its clear why the cast to any to bypass it is done.

Reply:
`as any` is not needed here, this was an artifact from our debugging session, I've removed it

Reply:
@bbangert I just tested with 
```
-    "@apollo/client": "^3.2.5",
+    "@apollo/client": "^3.3.9",
```
And with the new version we need `as any` here. This seems more desirable than the outdated dependency

Reply:
Possibly, it depends whether any of the other libraries checking on this object are going to do an `isinstance` check, if they do, the version mismatch will throw them as its not an instance of the same class.

Reply:
Updated with note explaining why we have `as any` here. And that it should be removed once, `apollo-upload-client` gets itself in sync with `apollo/client`

Reply:
if we do a `yarn up @apollo/client apollo-upload-client` then `yarn dedupe` everything will be on `3.3.9` ... assuming there's no unrelated breaking changes in other deps from the dedupe.